sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2Suite.scala

```diff
@@ -819,7 +819,7 @@ class DataSourceV2Suite extends QueryTest with SharedSparkSession with AdaptiveS
              |OPTIONS (PATH '$path')
              |AS VALUES (2, 3)
              |""".stripMargin)
-        checkAnswer(sql("SELECT * FROM test"), Seq(Row(0, 1), Row(0, 1), Row(1, 2), Row(2, 3)))
```

Comment:
RTAS with ds v2 source is newly supported in 4.0: https://github.com/apache/spark/pull/44190

It's not too late to change it as 4.0 is not released yet.